.bingo/README.md

```diff
@@ -1,14 +1,13 @@
-# Project Development Dependencies.
+# Binary dependencies
 
-This is directory which stores Go modules with pinned buildable package that is used within this repository, managed by https://github.com/bwplotka/bingo.
+This directory supports tools written in go, but used in `make`. Tools like `goimports` are built
+on-demand, and their dependencies do not affect the main project [go.mod](../go.mod).
 
-* Run `bingo get` to install all tools having each own module file in this directory.
-* Run `bingo get <tool>` to install <tool> that have own module file in this directory.
-* For Makefile: Make sure to put `include .bingo/Variables.mk` in your Makefile, then use $(<upper case tool name>) variable where <tool> is the .bingo/<tool>.mod.
-* For shell: Run `source .bingo/variables.env` to source all environment variable for each tool.
-* For go: Import `.bingo/variables.go` to for variable names.
-* See https://github.com/bwplotka/bingo or -h on how to add, remove or change binaries dependencies.
+The contents here were generated by [bingo](https://github.com/bwplotka/bingo) but have no runtime
+dependency on it in any way.
 
-## Requirements
+Integration in [Makefile](../Makefile) requires inclusion of the [Variables.mk](Variables.mk), then
+reference as a variable like `$(GOIMPORTS)`.
 
-* Go 1.14+
+This differs slightly from the defaults of the Bingo project:
+* ENV variable support is not included as this project only uses `make`
```

Comment:
I removed this and rewrote the README as it I think the result is more relevant than the generated README